plugins/gossip/parameters.go

```diff
@@ -11,6 +13,14 @@ type ParametersDefinition struct {
 
 	// MissingMessageRequestRelayProbability defines the probability of missing message requests being relayed to other neighbors.
 	MissingMessageRequestRelayProbability float64 `default:"0.01" usage:"the probability of missing message requests being relayed to other neighbors"`
+
+	MessagesRateLimit messagesLimitParameters
+}
+
+type messagesLimitParameters struct {
```

Comment:
I'd expect these parameter to be set in the firewall plugin and be named`firewall.rateLimiter.<parameter>` or something like this.

Reply:
I wanted to decouple firewall logic and rate limiters logic. We might have multiple different rate limiters with specific configuration settings that are injected in various goshimmer places (gossip, tangle, etc.). 
Other than that, not only a rate limiter is allowed to trigger the firewall, potentially it could be exposed via web API and triggered from the outside.
Even more, I tried to put the RateLimiter and Firewall in the single package in the begging and I got cycle dependency because Firewall depends on Gossip and Gossip depends on RateLimiter.